src/Model/Config.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tweakwise\TweakwiseJs\Model;
+
+use Magento\Framework\App\Config\ScopeConfigInterface;
+use Magento\Store\Model\ScopeInterface;
+
+class Config
+{
+    private const XML_PATH_ENABLED = 'tweakwise/tweakwisejs/general/enabled';
+    private const XML_PATH_INSTANCE_KEY = 'tweakwise/tweakwisejs/general/instance_key';
+
+    public const XML_PATH_MERCHANDISING_ENABLED = 'tweakwise/tweakwisejs/merchandising/enabled';
+
+    private const XML_PATH_SEARCH_TYPE = 'tweakwise/tweakwisejs/search/type';
+
+    /**
+     * @param ScopeConfigInterface $scopeConfig
+     */
+    public function __construct(
+        private readonly ScopeConfigInterface $scopeConfig
+    ) {
+    }
+
+    /**
+     * @return bool
+     */
+    public function isEnabled(): bool
+    {
+        return $this->scopeConfig->isSetFlag(self::XML_PATH_ENABLED, ScopeInterface::SCOPE_STORE);
+    }
+
+    /**
+     * @return string|null
+     */
+    public function getInstanceKey(): ?string
+    {
+        return $this->scopeConfig->getValue(self::XML_PATH_INSTANCE_KEY, ScopeInterface::SCOPE_STORE);
```

Comment:
Is deze instance key niet ook al instelbaar in de main tweakwise module?
Voelt een beetje duppelop nu qua configuratie.